src/app/tools/RulesContent.js

```diff
@@ -0,0 +1,8 @@
+export const RulesContent = (index) => {
+  const rulesContainer = document.querySelector('#rules p');
+  const rulesContent = ['You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options.', 'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select which vehicle from Star Wars is showed on the left from available options.', 'You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select which starship from Star Wars is showed on the left from available options.'];
+  rulesContainer.textContent = rulesContent[index];
+}
+
+
+
```

Comment:
Nie jestem przekonany do takiego rozwiązania, może zróbmy to jako dodatkową metode w klasie Rules, wtedy bedziemy miec wszystko w jednym mijescu. Jeżeli chciałbyś wyodrębnić coś to jedynie tą tablice z tekstami. Wtedy byśmy nazwali plik constants.js i w nim przechowywali wszystkie właśnie takie zmienne byśmy mieli wtedy
export const rulesContent = ...

Reply:
gotowe